src/components/HomeHeader.js

```diff
@@ -1,184 +1,200 @@
-          <div className='chart-header-logo-container'>
-            <img alt='org logo' src={this.props.icon} height='30px' />
+        <div className="chart-header-title-container">
+          <div className="chart-header-logo-container">
+            <img alt="org logo" src={this.props.icon} height="30px" />
           </div>
-          <div className='chart-header-title'>{this.props.title}</div>
+          <div className="chart-header-title">{this.props.title}</div>
         </div>
       );
     }
   };
 
   getHeaderHeight = () => {
-    let height = '300px';
+    let height = "300px";
     if (!this.props.showHomeText) {
-      height = '272.5px';
+      height = "272.5px";
     }
     return {
-      height
+      height,
     };
   };
 
-  handleSearchTermChange = evt => {
+  handleSearchTermChange = (evt) => {
     try {
       if (evt) {
         const searchTerm = evt.target.value;
         this.setState({ searchTerm });
         this.props.filterChartsOnSearch(searchTerm);
       }
     } catch (e) {
-      console.log('Catch error:', e);
+      console.log("Catch error:", e);
     }
   };
 
-  changeVersion = version => {
-    localStorage.setItem('version', version.value);
-    this.props.loadCharts(localStorage.getItem('version'));
-    this.props.history.push('/');
+  changeVersion = (version) => {
+    localStorage.setItem("version", version.value);
+    this.props.loadCharts(localStorage.getItem("version"));
+    this.props.history.push("/");
     window.location.reload();
-  }
+  };
 
   render() {
     try {
       return (
-        <div className='hero-bg'>
-          <div className='home-header-container' style={this.getHeaderHeight()}>
-            <div className='top-header-container'>
+        <div className="hero-bg">
+          <div className="home-header-container" style={this.getHeaderHeight()}>
+            <div className="top-header-container">
               <a
-                href='https://litmuschaos.io'
-                target='_blank'
-                rel='noopener noreferrer'
+                href="https://litmuschaos.io"
+                target="_blank"
+                rel="noopener noreferrer"
               >
-                <div className='top-header-divide-container'>
+                <div className="top-header-divide-container">
                   <img
-                    alt='organization logo'
-                    src={process.env.PUBLIC_URL + '/icons/litmus.png'}
-                    className='logo-img'
+                    alt="organization logo"
+                    src={process.env.PUBLIC_URL + "/icons/litmus.png"}
+                    className="logo-img"
                   />
                 </div>
               </a>
-              <Link to={process.env.PUBLIC_URL} className='home'>
-                <h3 className='top-header-home'>{<FaHome />}</h3>
+              <Link to={process.env.PUBLIC_URL} className="home">
+                <h3 className="top-header-home">{<FaHome />}</h3>
```

Comment:
Please, resize home icon make it some large

Reply:
Ok

Reply:
Done. Submitting new PR and adding some screenshots.